cpp/daal/src/algorithms/k_nearest_neighbors/kdtree_knn_classification_train_dense_default_impl.i

```diff
@@ -189,9 +238,7 @@ Status KNNClassificationTrainBatchKernel<algorithmFpType, training::defaultDense
     typedef BoundingBox<algorithmFpType> BBox;
 
     const algorithmFpType base = 2.0;
-    // The queue size is not impacted by number of threads.
-    // All operations with the queue are done not in the threader_for primitives.
-    const size_t queueSize = 2 * Math::sPowx(base, Math::sCeil(Math::sLog(__KDTREE_FIRST_PART_LEAF_NODES_PER_THREAD) / Math::sLog(base)));
+    const size_t queueSize     = 2 * Math::sPowx(base, Math::sCeil(Math::sLog(__KDTREE_FIRST_PART_LEAF_NODES_PER_THREAD) / Math::sLog(base)));
```

Comment:
Add comment about the meaning of that value: (seems it is the lowest power of two that is greater than log_2(__KDTREE_FIRST_PART_LEAF_NODES_PER_THREAD) multiplied by 2)